Yoast/wordpress-seo
# Add a notification to the analyses when the Recalibration Beta is activated.

## Summary

This PR can be summarized in the following changelog entry:

* Adds a notification with a help text to the analyses when the Recalibration Beta is activated. 

## Relevant technical choices:

* Moved the HelpLink component to its own file, out of SeoAnalysis.js.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch. 
* Go to `SEO-> General -> Features` at the bottom under "Get an even better analysis" activate the beta with the toggle. 
* Open or create a new post. See that behind the "Readability Analysis" and "Focus Keyphrase" it now reads "( beta )" in both the Sidebar as well as the metabox.
* Open the tabs: Right at the top "Recalibration beta is active" should appear, which is clickable and should point to the correct link. 
* Go back to the beta toggle and turn it off again. The notifications should disappear. 

## Quality assurance


Fixes #11606
